simcon_project/users/migrations/0005_auto_20210103_0336.py

```diff
@@ -0,0 +1,102 @@
+# Generated by Django 3.1.3 on 2021-01-03 03:36
```

Comment:
remove from PR, any migration files should not be pushed up, according to git ignore